models/uniswap/avalanche_c/uniswap_avalanche_c_schema.yml

```diff
@@ -0,0 +1,96 @@
+version: 2
+
+models:
+  - name: uniswap_v3_avalanche_c_trades
+    meta:
+      blockchain: avalanche_c
+      sector: dex
+      project: uniswap_v3
+      contributors: phu
+    config:
+      tags: ['avalanche_c','uniswap_v3','trades', 'uniswap','dex']
+    description: >
+        Uniswap V3 contract trades on Avalanche_c
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+      - check_dex_seed:
+          blockchain: avalanche_c
+          project: uniswap
+          version: 3
```

Comment:
Hi @lequangphu, in order for this `check_dex_seed` test to work,
please add some seed records in this file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

Reply:
sure, let me add, thanks!

Reply:
the records should be added in the same PR with the model, right?

Reply:
Yes.